### ISSUE TITLE
Python 3.5 wheels?

### ISSUE THREAD

**Issue by rly** [PyTables/PyTables#773]
Since there are no wheel files for Python 3.5 on PyPI for the latest PyTables 3.6.0, trying to install PyTables on Python 3.5 requires building the wheel, which results in the below error on Windows.

Similarly, because there are no wheel files for Python 3.5, hdf5 is not being bundled, and I get the below error on Linux.

Could wheels for Python 3.5 be provided? Thanks.

## Windows
```bash
conda create --name py35 python=3.5
pip install tables
```
resulted in lots of output ending in:
```
LINK : fatal error LNK1181: cannot open input file 'Files\HDF_Group\HDF5\1.10.5\lib.obj'
    error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\BIN\\x86_amd64\\link.exe' failed with exit status 1181
```
Both building the wheel and running setup.py resulted in the above output. 
1. It seems like the space in the path to lib.obj is not properly escaped.
2. Even if it is properly escaped, my filesystem does not have the file C:\Program Files\HDF_Group\HDF5\1.10.5\lib.obj

Env:
Windows 10 with Microsoft Visual Studio 14.0 Build Tools and HDF5 1.10.5 installed.
Miniconda for Python 3.7. Conda installed Python 3.5.6. 

This was not an issue with the previous release of PyTables, for which Python 3.5 wheels were built, and is not an issue with the current release of PyTables with Python 3.6 and 3.7. 

## Linux (on CircleCI)
```
Collecting tables>=3.5.1
  Downloading https://files.pythonhosted.org/packages/fb/b6/f7128895d328f64e52f61fbb39c03a458f8b583724705bddae392a1085a0/tables-3.6.0.tar.gz (7.8MB)
    ERROR: Command errored out with exit status 1:
     command: /home/circleci/project/.tox/gallery-py35/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-lu4c0l_r/tables/setup.py'"'"'; __file__='"'"'/tmp/pip-install-lu4c0l_r/tables/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-lu4c0l_r/tables/pip-egg-info
         cwd: /tmp/pip-install-lu4c0l_r/tables/
    Complete output (8 lines):
    /usr/bin/ld: cannot find -lhdf5
    collect2: error: ld returned 1 exit status
    * Using Python 3.5.7 (default, Jun 19 2019, 02:10:09)
    * USE_PKGCONFIG: True
    .. ERROR:: Could not find a local HDF5 installation.
       You may need to explicitly state where your local HDF5 headers and
       library can be found by setting the ``HDF5_DIR`` environment
       variable or by using the ``--hdf5`` command-line option.
```

This was not an issue with the previous release of PyTables, for which Python 3.5 wheels were built, and is not an issue with the current release of PyTables with Python 3.6 and 3.7.

**Comment by rly** [comment]
I see. Thanks for letting me know!

**Comment by tomkooij** [comment]
I fixed the python 3.5 wheels, will release py35 windows wheels for 3.6.1

**Comment by tomkooij** [comment]
py35 windows wheels now at PyPI
